public/lib/helpers/timeline_helper.js

```diff
@@ -26,6 +30,28 @@ define(function (require) {
       }
     };
 
+    TimelineHelper.prototype.changeTimezone  = function ($scope) {
```

Comment:
Yes, and that's while I'm still working on this aspect... Maybe I can return the _utcOffset_ and set the `$scope` not in function?